.github/workflows/lint-code.yml

```diff
@@ -33,5 +33,32 @@ jobs:
           python3 -m pip install --upgrade pip
           pip3 install flake8
 
+      - name: 'Install Swift dependencies'
+        shell: bash
+        run: |
+          set -e
+  
+          curl -LO https://swiftlang.github.io/swiftly/swiftly-install.sh
```

Comment:
😬 swift-format is only packaged on homebrew and Arch, apparently.

https://repology.org/project/swift-format/versions

And nixpkgs, but they're on an old version.

Reply:
... in light of this, should I just move the lint job to a macOS runner? then we could brew install instead of worrying about swiftly (yet)

e.g 
https://github.com/hylo-lang/hylo/blob/main/.github/workflows/format-check.yml#L32C13-L33C26

Reply:
yes, but now I have to download gn from chromium infra packages :upside_down_face: 

Just can't win.